src/templates/post.html

```diff
@@ -1,19 +1,23 @@
-{% if post.resto == 0 %}
+{% if post.resto == 0 and not search_result%}
+<div class="thread-stats">{% if post.sticky %}Sticky / {% endif %}{% if post.closed %}Closed / {% endif %}<span class="ts-replies" data-tip="Replies" title="Replies">{{post.replies}}</span> / <span class="ts-images" data-tip="Images" title="Files">{{post.images}}</span> / <span data-tip="Posters" class="ts-ips" title="Posters">{% if post.posters %}{{post.posters}}{% else %}?{% endif %}</span></div>
     <div class="postContainer opContainer" id="pc{{post.no}}">
 {% else %}
     <div class="postContainer replyContainer" id="pc{{post.no}}">
+	<div class="sideArrows" id="sa{{post.no}}">&gt;&gt;</div>
 {% endif %}
 
-    <div class="sideArrows" id="sa{{post.no}}">&gt;&gt;</div>
 
-    {% if post.resto == 0 %}
-        <div id="p{{post.no}}" class="post op reply">
```

Comment:
Removing the `reply` might break search formatting. I remember add it there to fix it. I will have to test this